scripts/override_os_images.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/local/bin/python3
```

Comment:
to the best of my knowledge, this is superior as it's less coupled to specific locations of the python binary (which can change based on installation method) and is equivalent to doing ``python3`` in bash:
```suggestion
#!/usr/bin/env python3
```